src/components/ScreenWrapper/index.native.js

```diff
@@ -0,0 +1,174 @@
+import {Keyboard, View, PanResponder} from 'react-native';
+import React from 'react';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import DevMenu from 'react-native-dev-menu';
+import KeyboardAvoidingView from '../KeyboardAvoidingView';
+import CONST from '../../CONST';
+import KeyboardShortcut from '../../libs/KeyboardShortcut';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import HeaderGap from '../HeaderGap';
+import OfflineIndicator from '../OfflineIndicator';
+import compose from '../../libs/compose';
+import withNavigation from '../withNavigation';
+import withWindowDimensions from '../withWindowDimensions';
+import withEnvironment from '../withEnvironment';
+import ONYXKEYS from '../../ONYXKEYS';
+import {withNetwork} from '../OnyxProvider';
+import {propTypes, defaultProps} from './propTypes';
+import SafeAreaConsumer from '../SafeAreaConsumer';
+import TestToolsModal from '../TestToolsModal';
+import withKeyboardState from '../withKeyboardState';
+import * as App from '../../libs/actions/App';
+
+class ScreenWrapper extends React.Component {
```

Comment:
I see a lot of code duplication between the `index` and `index.native.js`, could we please make this code DRY?